packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h

```diff
@@ -0,0 +1,426 @@
+          secondsModifiedSinceEpoch:(NSNumber *)secondsModifiedSinceEpoch
+                         completion:
+                             (void (^)(NSNumber *_Nullable, FlutterError *_Nullable))completion;
+@end
+
+extern void FWFWKWebsiteDataStoreHostApiSetup(
+    id<FlutterBinaryMessenger> binaryMessenger,
+    NSObject<FWFWKWebsiteDataStoreHostApi> *_Nullable api);
+
+/// The codec used by FWFUIViewHostApi.
+NSObject<FlutterMessageCodec> *FWFUIViewHostApiGetCodec(void);
+
+@protocol FWFUIViewHostApi
+/// @return `nil` only when `error != nil`.
+- (nullable NSArray<NSNumber *> *)getContentOffsetInstanceId:(NSNumber *)instanceId
+                                                       error:
+                                                           (FlutterError *_Nullable *_Nonnull)error;
+- (void)setBackgroundColorInstanceId:(NSNumber *)instanceId
+                               value:(nullable NSNumber *)value
+                               error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)setOpaqueInstanceId:(NSNumber *)instanceId
+                     opaque:(NSNumber *)opaque
+                      error:(FlutterError *_Nullable *_Nonnull)error;
+@end
+
+extern void FWFUIViewHostApiSetup(id<FlutterBinaryMessenger> binaryMessenger,
+                                  NSObject<FWFUIViewHostApi> *_Nullable api);
+
+/// The codec used by FWFUIScrollViewHostApi.
+NSObject<FlutterMessageCodec> *FWFUIScrollViewHostApiGetCodec(void);
+
+@protocol FWFUIScrollViewHostApi
+- (void)createFromWebViewInstanceId:(NSNumber *)instanceId
+                  webViewInstanceId:(NSNumber *)webViewInstanceId
+                              error:(FlutterError *_Nullable *_Nonnull)error;
+/// @return `nil` only when `error != nil`.
+- (nullable NSArray<NSNumber *> *)getContentOffsetInstanceId:(NSNumber *)instanceId
+                                                       error:
+                                                           (FlutterError *_Nullable *_Nonnull)error;
+- (void)scrollByInstanceId:(NSNumber *)instanceId
+                         x:(NSNumber *)x
+                         y:(NSNumber *)y
+                     error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)setContentOffsetInstanceId:(NSNumber *)instanceId
+                                 x:(NSNumber *)x
+                                 y:(NSNumber *)y
+                             error:(FlutterError *_Nullable *_Nonnull)error;
+@end
+
+extern void FWFUIScrollViewHostApiSetup(id<FlutterBinaryMessenger> binaryMessenger,
+                                        NSObject<FWFUIScrollViewHostApi> *_Nullable api);
+
+/// The codec used by FWFWKWebViewConfigurationHostApi.
+NSObject<FlutterMessageCodec> *FWFWKWebViewConfigurationHostApiGetCodec(void);
+
+@protocol FWFWKWebViewConfigurationHostApi
+- (void)createInstanceId:(NSNumber *)instanceId error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)createFromWebViewInstanceId:(NSNumber *)instanceId
```

Comment:
Same naming problem here as earlier, where the first instance has the wrong name.